pkg/gvks/supportedgvks/supportedgvks.go

```diff
@@ -39,6 +40,8 @@ func ManualResources(smLoader *servicemappingloader.ServiceMappingLoader, servic
 func resources(smLoader *servicemappingloader.ServiceMappingLoader, serviceMetaLoader metadata.ServiceMetadataLoader, includesAutoGen bool) []schema.GroupVersionKind {
```

Comment:
Mind explaining when the supportedgvk is called?

Reply:
It is called at multiple places, like crd generation, resource doc generation, cnrm role generation, and etc, when the caller needs to know what all the GVKs are, what manual GVKs are.